src/Kunstmaan/MediaBundle/Helper/File/SVGMimeTypeGuesser.php

```diff
@@ -31,14 +31,14 @@ public function guess($path)
             throw new AccessDeniedException($path);
         }
 
-        if (!self::isSupported()) {
-            return;
+        if (!$this->isGuesserSupported()) {
```

Comment:
Why is this changes needed? Otherwise I would revert it, same for the return null's